stdlib/universe/join.go

```diff
@@ -808,6 +828,23 @@ func (c *MergeJoinCache) buildPostJoinSchema() {
 	}
 }
 
+// equalJoinKeys compares two keys for equality.
+// Null values are not considered equal when joining (unlike when grouping).
+func equalJoinkeys(left, right flux.GroupKey) bool {
+	for _, v := range left.Values() {
+		if v.IsNull() {
+			return false
+		}
+	}
+	for _, v := range right.Values() {
+		if v.IsNull() {
```

Comment:
Is this necessary after you added the `isNull()` check to `values.Equal()`? That is, can you reduce this function to just `left.Equal(right)`?

Reply:
Yes, you're right.  I'll fix it.